corehq/motech/repeaters/models.py

```diff
@@ -1476,6 +1476,7 @@ def requeue(self):
         elif self.state == State.Cancelled:
             self.state = State.Fail
         self.next_check = datetime.utcnow()
+        self.max_possible_tries = self.num_attempts + MAX_BACKOFF_ATTEMPTS
```

Comment:
I may have let it slip by in previous commits, but this seems like a fundamental change to processing logic. I'm surprised to see `max_possible_tries` be updated at all, let alone to a value that seems rather dynamic (`self.num_attempts + constant`). Apologies if I missed this in a previous commit, but can you elaborate on this?

Reply:
This was done to allow more than one retry when `requeue` is called. It's implemented differently in SQL because the number of attempts is based on `len(record.attempts)` where in Couch it's based on the `overall_tries` field, which is incremented on each attempt and reset to zero on `requeue`.

So this is actually keeping the processing logic the same, not changing it.